DFC.Api.AppRegistry/Models/Legacy/LegacyRegionModel.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using DFC.Api.AppRegistry.Enums;
+using Microsoft.Azure.Documents;
 using System;
 using System.Diagnostics.CodeAnalysis;
 
 namespace DFC.Api.AppRegistry.Models.Legacy
 {
     [ExcludeFromCodeCoverage]
-    public class LegacyRegionModel
+    public class LegacyRegionModel : Document
```

Comment:
Added this to make the CosmosDBRegion trigger testable, I can't see any obvious side effects?